src/main/java/uk/ac/cam/cl/dtg/segue/api/AdminFacade.java

```diff
@@ -281,6 +294,112 @@ public synchronized Response modifyUsersRole(@Context final HttpServletRequest r
     return Response.ok().build();
   }
 
+  /**
+   * This method will allow users to have their teacher pending status mass changed.
+   *
+   * @param request - to help determine access rights.
+   * @param status    - new teacher pending status.
+   * @param userIds - a list of user ids to change en-mass
+   * @return Success shown by returning an ok response
+   */
+  @POST
+  @Path("/users/teacher_pending/{status}")
```

Comment:
There are other endpoints under `/admin/` with names such as:
- `/admin/users/change_email_verification_status/{emailVerificationStatus}/{checkEmailsExistBeforeApplying}`
- `/admin/users/change_role/{role}`

They're not great names, but perhaps we can rename this one so that it's in line with those. Something like `/users/change_teacher_pending/{status}`.

None of the options are very RESTFul, unfortunately.

What do you think?

Reply:
Yeah that makes sense!